components/textField.tsx

```diff
@@ -0,0 +1,14 @@
+const title = "Student Bio";
+
+// import { AppProps } from "next/dist/next-server/lib/router/router";
```

Comment:
what are these commented out imports for AppProps for? Should we preserve them for something later?